### PR TITLE
setup pulumi sample for GraphQL endpoint in AppSync & DynamoDB

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: appsync-dynamodb-endpoint/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Start LocalStack via:
+
+```bash
+localstack start -d
+```
+
+Create a new Pulumi stack via:
+
+```bash
+pulumilocal stack init python-eks-testing
```
Comment (whummer) [inline appsync-dynamodb-endpoint/README.md:23]: nit: could be renamed to `python-appsync-testing` or similar..

File: appsync-dynamodb-endpoint/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    key     : [secret]
+
+Resources:
+    + 11 created
+
+Duration: 20s
+```
+
+You can retrieve the GraphQL endpoint API key via the [AppSync Resource Browser](https://app.localstack.cloud/resources/appsync) on the LocalStack Web Application. You can test the deployed GraphQL endpoint via:
+
```
Comment (whummer) [inline appsync-dynamodb-endpoint/README.md:54]: nit: maybe we could also add a sample command+output here to retrieve the API key via the CLI? (via `awslocal appsync list-api-keys ...`)

File: appsync-dynamodb-endpoint/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    -H "x-api-key:<API_KEY>" -d '{ "query": "mutation AddTenant { addTenant(id: \"123\", name: \"FirstCorp\") { id name } }" }' \
+    <ENDPOINT_URL>
+```
+
+Replace the `<API_KEY>` and `<ENDPOINT_URL>` placeholders with the values from the Pulumi stack output and the LocalStack Web Application. You should see the following response:
+
+```bash
+{"data": {"addTenant": {"id": "123", "name": "FirstCorp"}}}
+```
+
```
Comment (whummer) [inline appsync-dynamodb-endpoint/README.md:66]: nit: we could also add a command to retrieve an item via the `getTenantById(..)` operation, for additional illustration purposes..

**Review by whummer — APPROVED** [review]
Nice sample @HarshCasper ! 🚀 Added a couple of minor suggestions, but nothing critical..

Also /cc @simonrw who may have some additional feedback..

File: appsync-dynamodb-endpoint/README.md (hunk 3 above)
Comment (HarshCasper) [reply appsync-dynamodb-endpoint/README.md:66]: Will look into it in a second iteration

**Pull request merged**